Automattic/jetpack
# Node: match version used by WordPress itself

In https://core.trac.wordpress.org/ticket/59663, WordPress Core jumped from being pretty far behind on the LTS version of Node it required to 20.x, and specifically 20.10. To simplify things since we're close, what about us upping a tad to match?

Separately, would need to consider this for Calypso, but wanted to run it here for opinions first.

## Proposed changes:

* Update nvmrc to 20.10 from 20.8.1
* Update package.json for the proper engine.

### Other information:


## Jetpack product discussion


None as of now.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Run `nvm use` to update to 20.10 (possibly need `nvm install` first).
* Run pnpm install, jetpack install, jetpack build, etc to ensure things build right.
* Verify CI is happy.
